views/home.dt

```diff
@@ -1,117 +1,75 @@
+		project name now which you can start any time. #[dub run] recompiles your
+		program if neccessary and runs it afterwards. You can now add dependencies
+		to your dub.json/dub.sdl and simply import them from code. DUB will handle
+		the rest. 
+	a(href="getting_started")
+		button Read Further
 
-	table
-
-		- static immutable browse_kinds = ["name;Name", "updated;Last update", "rating;Rating", "added;Registered", ";Description"];
-		tr
-			- foreach (i, c; browse_kinds)
-				- auto cp = c.split(";");
-				- if (cp[0] == "rating" && "showrating" !in req.query)
-					- continue;
-				- if (cp[0].length == 0)
-					th= cp[1]
-				- else if (sort_mode == cp[0])
-					th.selected= cp[1]
-				- else
-					th
-						a(href=makeHomeURL(cp[0], category, info.skip, info.limit))= cp[1]
-
-		- foreach (pl; info.packages)
-			- if( pl["versions"].length )
-				- auto vidx = getBestVersionIndex(pl["versions"].get!(Json[]).map!(v => v["version"].opt!string));
-				- auto p = pl["versions"][vidx];
-				- auto desc = p["description"].opt!string;
-				- auto ver = p["version"].opt!string();
-				tr
-					td
-						- string image;
-						- if (pl["categories"].length)
-							- if (auto pc = pl["categories"][0].get!string in info.categoryMap)
-								- image = pc.imageName;
-						- if (!image.length) image = "unknown";
-						img.category-icon(src="images/categories/#{image}.png", title=image.replace(".", "/"))
-						a(href='packages/#{pl["name"].get!string}')= pl["name"].get!string
-					td.nobreak
-						- if (ver.length <= 20)
-							|= ver
-						- else
-							| #{ver[0 .. 18]}&hellip;
-						span.dull.nobreak(title=formatDateTime(p["date"]))<> , #{formatFuzzyDate(p["date"])}
-					- if ("showrating" in req.query)
-						td.nobreak(title="#{formatPackageStats(pl.stats)}", style="color: #B03931;")= formatRating(pl.stats.rating)
-					td.nobreak(title=formatDateTime(pl["dateAdded"]))= formatDate(pl["dateAdded"])
-					- if (desc.length <= 100)
-						td= desc
-					- else
-						td #{desc[0 .. 98]}&hellip;
-
-	ul.pageNav
-		- foreach (i; 0 .. (info.packageCount + info.limit - 1) / info.limit)
-			- if (i * info.limit == info.skip)
-				li.selected= i+1
-			- else
-				li: a(href=makeHomeURL(sort_mode, category, i*info.limit, info.limit))= i+1
-	ul.pageNav.perPage
-		- static immutable limit_enum = [10, 20, 50, 100];
-		- foreach (limit; limit_enum)
-			- if (info.limit == limit)
-				li.selected= limit
-			- else
-				li: a(href=makeHomeURL(sort_mode, category, info.skip, limit))= limit
-
-	p Displaying results #{info.skip+1} to #{min(info.skip+info.limit+1, info.packageCount)} of #{info.packageCount} packages found.
-
-	- if (!mirror.length)
-		- if (req.session)
-			p: a(href="my_packages") Manage my packages
-		- else
-			p Please #[a(href="login?redirect=/my_packages") log in] to manage your own packages.
+	h2 Browse Packages
+	h3 Best Rated Packages
+		.packageHome
+	h3 Recently Updated Packages
+		.packageHome
```

Comment:
yes this was planned to be added. Currently I didn't work on it a lot because I can't run dub-registry anymore after a mongodb update. I made a PR to vibe.d but it still needs a review. I think I will make it work using my fork when I have time again on weekend next week